### PR TITLE
ISSUE-2640: BP-43: github actions: Run bookie-tests using gradle

### PR DESCRIPTION
**Pull request by sursingh**
### Motivation

Migrate bookkeeper build to gradle. 

### Changes

Adapt github actions to use gradle for running bookie tests

Master Issue: #2640

### CODE REVIEW

**Comment by sursingh** [comment]
rerun failure checks

**Comment by sursingh** [comment]
rerun failure checks

**Comment by sursingh** [comment]
rerun failure checks

File: bookkeeper-server/build.gradle [hunk 1]
```diff
@@ -64,9 +66,13 @@ dependencies {
     testImplementation depLibs.junit
     testImplementation depLibs.junitFoundation
     testImplementation depLibs.kerbySimpleKdc
+    testImplementation depLibs.metricsCore
     testImplementation depLibs.mockito
     testImplementation depLibs.powermockJunit
     testImplementation depLibs.powermockMockito
+    testImplementation depLibs.slf4j
+    testImplementation depLibs.slf4jLog4j
```
Comment (pkumar-singh) [inline bookkeeper-server/build.gradle:74]: If its already implementation dependency it does not have to be testImplementation dependency.

**Pull request closed without merging**